tests/src/Kernel/TestFrameworkKernelTest.php

```diff
@@ -109,10 +109,10 @@ public function testIntegrationToggle() {
       ->getConfig('handler');
 
     if ($this->integration_enabled) {
-      self::assertTrue(($handler instanceof HandlerStack));
+      self::assertInstanceOf(MockHandler::class, $handler);
```

Comment:
This looks good. All tests green now.

Reply:
This wrong and failing for me. If integration is enabled, the mock handler should  be disabled. That means the handler should be `HandlerStack::class` so we have these backwards.